src/main/java/openevent/model/api/ApiSponsor.java

```diff
@@ -0,0 +1,109 @@
+package openevent.model.api;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+@Type("sponsor")
+public class ApiSponsor {
+	
+	@Id
+	@JsonProperty
```

Comment:
@iamareebjamal 
what is the use of this class?
we are using sponser.java as a model right, so the object sponser will be of this type and we will take data through that.
what exactly does this class do?